made first draft for data

emily-shaw opened the pull request.

rganesh1234567 reviewed (commented):
shouldn't health, attack and defense be numbers? Since they are numeric stats 
what is the difference between "image" and "character_image". Unless there is a specific reason, you could just simplify it into one image field. 

comments and question for reviewers
Yes, including a size attribute for characters and objects to be larger than 1x1 tiles is important for flexibility in map design. This already seems partially addressed with character_size and object_size.
It might be useful to think about state tracking for map tiles, such as whether they’re occupied, passable, or blocked.
As far as drawing goes, I think we can wait till the UI diagram file is done

emily-shaw commented:
I have updated the type for health, attack and defense, deleted image and kept "character_image". And I added 
"tile_sate" to see if the tile is currently being used. It should be good to merge now and I'll add Drawing in the final version of data.md once UI is finished
